### PR TITLE
Fix having black background when it should be transparent

### PR DESCRIPTION
**Pull request by pajong**
### This PR will...
Put transparent background to the newly-added .jw-wrapper class

### Why is this Pull Request needed?
jw-wrapper caused a regression of having black background, when it should be transparent.

#### Addresses Issue(s):

JW8-###

### CODE REVIEW

**Comment by johnBartos** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  🛠 There are modified src files, but no test changes. Add tests if you're able to.
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  🗿 Set a milestone. It should be the ticket's fix version in JIRA.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/css/jwplayer/imports/jwplayerlayout.less [hunk 1]
```diff
@@ -30,6 +30,10 @@
         }
     }
 
+    .jw-wrapper {
```
Comment (vseventer) [inline src/css/jwplayer/imports/jwplayerlayout.less]: Remove background-color from `.jw-player` above, and move it to line 99 below.

File: src/css/jwplayer/states.less [hunk 2]
```diff
@@ -4,6 +4,10 @@
 .jw-state-setup {
     background-color: transparent;
 
+    .jw-wrapper {
+        background-color: transparent;
```
Comment (robwalch) [inline src/css/jwplayer/states.less]: Should it always be transparent? (not just on setup). Please compare the current Views page in this build (http://player-pr-test-jenkins.longtailvideo.com/builds/17782/archive/test/view/) with an earlier version before the wrapper was added.

Comment (pajong) [reply src/css/jwplayer/states.less]: The wrapper is used for floating player, and it needs the black background for floating player.
Without the background, there is no black space when there needs to be.

**Comment by pajong** [comment]
@robwalch Compared all scenario between this build (http://player-pr-test-jenkins.longtailvideo.com/builds/17793/archive/test/view/) and the build that you provided, and confirmed that all of them are matching.

File: src/css/jwplayer/states.less (hunk 2 above)
Comment (robwalch) [reply src/css/jwplayer/states.less]: But it is also present when we're not in floating mode, so the black background is applied everywhere except for setup. 

`.jw-wrapper { background-color: transparent; } ` should be a default (in jwplayerlayout not here in state-setup) setting and only make it black when floating.

**Pull request merged**